tl.lua

```diff
@@ -9850,6 +9850,15 @@ end
 
 
 
+local BOM = "\xEF\xBB\xBF"
+local function readFullFile(fd)
```

Comment:
minor coding style nitpick:

```suggestion
local function read_full_file(fd)
   local bom <const> = "\xEF\xBB\xBF"
```

Could you change it to use "snake_case" instead of "methodCase" for consistency with the rest of the file? Also, it should be fine to move the BOM constant inside the file and make it lowercase as well.

Reply:
Done